src/ship.js

```diff
@@ -0,0 +1,9 @@
+const Ship = (name, shipLength) => {
+  const position = new Array(shipLength).fill('');
+  const hitPositions = [];
+
+  return {
+    name, shipLength, position, hitPositions,
+  };
+};
+export default Ship;
```

Comment:
- this class should be able to tell if ship has been sunk [[ see requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/battleship/sunk_requirement.png) ]

- this should have a `hit()` method that takes a number as an argument and marks that position as hit [[see requiredments](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/battleship/hit_requirement.png)]

Reply:
isSunk() and hit() functions were successfully implemented.